### PR TITLE
JSON Payment Protocol V2 - Transmit Signed Tx - Multiple Outputs

### PR DESCRIPTION
**Pull request by stevenzeiler**
#### PR Requirements

If you have made **any** visual changes to the GUI. Make sure you have:
- [ ] Tested on iOS Tablet
- [ ] Tested on small Android
- [ ] n/a

### CODE REVIEW

File: src/actions/BitPayActions.js [hunk 1]
```diff
@@ -147,23 +147,19 @@ export async function launchBitPay(
...
   const invoiceInstruction = invoiceResponse.instructions[0]
   errorData = { ...errorData, invoiceInstruction }
-  if (invoiceInstruction.outputs) {
-    if (invoiceInstruction.outputs.length > 1) {
-      throw new BitPayError('MultiOutputInvoice', { errorData })
-    }
-  } else throw new BitPayError('EmptyOutputInvoice', { errorData })
-  const instructionOutput = invoiceInstruction.outputs[0]
-
+  if (!invoiceInstruction.outputs) {
```
Comment (paullinator) [inline src/actions/BitPayActions.js:150]: Need to check `outputs.length > 1`

**Review by paullinator — CHANGES_REQUESTED** [review]

File: src/actions/BitPayActions.js (hunk 1 above)
Comment (stevenzeiler) [reply src/actions/BitPayActions.js:150]: Do you mean > 0?

Comment (paullinator) [reply src/actions/BitPayActions.js:150]: Ah yes

Comment (stevenzeiler) [reply src/actions/BitPayActions.js]: Okay it now checks for an empty array of outputs.

File: src/actions/BitPayActions.js [hunk 2]
```diff
@@ -225,6 +221,20 @@ export async function launchBitPay(
...
+        fetchBitPayJsonResponse(uri, {
+          method: 'POST',
+          headers: {
+            'Content-Type': 'application/payment',
+            'x-paypro-version': '2'
+          },
+          body: JSON.stringify({
+            chain: requestCurrencyCode,
+            currency: requestCurrencyCode,
+            transactions: [{ tx: signedHex, weightedSize: signedHex.length / 2 }]
```
Comment (stevenzeiler) [inline src/actions/BitPayActions.js:234]: It appears that the `signedHex` which is transmitted here results in a different txid than the one Edge displays in the payment history.

That could possibly be the cause of the tx-mempool-conflict I am receiving when pushing transactions to the blockchair API.

**Review by paullinator — APPROVED** [review]

**Pull request merged**